ci-operator/step-registry/kueue-operator/image-patch/kueue-operator-image-patch-commands.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
+if [[ "$JOB_TYPE" == *"periodic"* ]]; then
```

Comment:
In reharshel jobs, the value of JOB_TYPE gets modified to presubmit, even if the job is a periodic one.
details here https://github.com/openshift/ci-tools/blob/main/pkg/rehearse/jobs.go#L559
Once this PR is merged and the periodic jobs run, we are expected to see the JOB_TYPE with value periodic and this step skipped.